css-three-column-layout/styles.css

```diff
@@ -0,0 +1,59 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(22 151 206);
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one {
+  flex-basis: 100%;
+}
+
+.column-third {
+  flex-basis: calc(100%/3);
+  background-color: white;
+  border-radius: 250px;
+  padding: 40px;
+  margin: 60px;
+}
+
+.character {
+  width: 100%;
+}
+
+.three-columns {
```

Comment:
since these columns should be the same size as the ones for the image, I would use the same class `column-third` for them